controllers/event/index.js

```diff
@@ -144,11 +150,13 @@ export const getAllEvents = async (req, res) => {
 };
 
 export const updateEvents = async (req, res) => {
+  let temp;
   const { email } = req.organizer;
   const { slug } = req.params;
+  const where = { paymentMethod: 'free' };
```

Comment:
`where` wouldn't be a good variable name. If the codebase grows bigger or someone new joins the team. This could be any issue and may cost the development in future.

Reply:
worked on this